compute_tools/src/spec.rs

```diff
@@ -562,7 +571,7 @@ pub fn handle_databases(spec: &ComputeSpec, client: &mut Client) -> Result<()> {
                 DatabaseAction::Create => " -> create",
                 DatabaseAction::Update => " -> update",
             };
-            info!("   - {}:{}{}", db.name, db.owner, action_str);
+            info!(" - {}:{}{}", db.name, db.owner, action_str);
```

Comment:
Maybe leave the two spaces in so it looks like an indented list? Or maybe add `\t`? Or should we accumulate all of these into a vector as well?